### PR TITLE
Add Jupyter Notebook for EBRAINS with down-scaled MAM

### PR DESCRIPTION
**Pull request by jhnnsnk**
The notebook illustrates the simulation workflow with a down-scaled version of the multi-area model.
It is based on a draft by @terhorstd.
I have added the functionality to write a custom `config.py` from within the notebook such that no additional files are necessary.
Besides, I have computed the instantaneous firing rates from the spike data and plotted them. Please check if the calculation is correct. Currently the mean-field prediction gives about 30 spikes / s and the simulation about 17 spikes / s and I am not sure if this discrepancy is expected.
I have also added some documentation on running the notebook in the Jupyter Lab on EBRAINS.

On EBRAINS the SLN fit failed due to an IndexError, but locally on my laptop it worked; so I assume that it is an issue with some Python package version. I have added the IndexError to the exception such that if it occurs the hard-coded fit parameters are used in the same way as when R is not available for calculation.

### CODE REVIEW

File: multi-area-model.ipynb [hunk 1]
```diff
@@ -0,0 +1,260 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "e940bb6b",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "scale_down_to = 0.005"
```
Comment (jarsi) [inline multi-area-model.ipynb:88]: Here you set the number of neurons and indegrees to .5 %, but above you state you would set it to 1%. Changing either one should be fine.

**Review by jarsi — CHANGES_REQUESTED** [review]
Thanks for doing this! I followed your explanation and ran it on the collab. It worked and ran really smoothly. Besides a small inconsistency in `scale_down_to` all looks nice.

In a future iteration one could enhance the analysis a bit, e.g. show how to extract spikes that belong to an area and plot them in a population resolved rasterplot.

File: multi-area-model.ipynb (hunk 1 above)
Comment (jhnnsnk) [reply multi-area-model.ipynb:88]: Thanks for pointing me to this mistake. It is fixed now.

**Review by jarsi — APPROVED** [review]

**Comment by jarsi** [comment]
Thanks for adding this new feature, I'll go ahead and merge.

**Pull request merged**